df_auth/settings.py

```diff
@@ -3,7 +3,7 @@
 
 
 DEFAULTS = {
-    "USER_IDENTITY_FIELDS": ("email",),
+    "USER_IDENTITY_FIELDS": ("email","username"),
```

Comment:
is this still needed?

Reply:
No, it's not required for social auth. I added it to get the access token with a username and password as I didn't succeed to do that with email and password. I guess authenticate function works with username and password. 🤔